Fix broken documentation links

spantaleev opened the pull request.

spantaleev committed:
Fix broken documentation links

MAGICCC commented:
Ah it was in my todo list already :p  
You could also edit the other files if you want
https://github.com/mailcow/mailcow-dockerized/search?q=https%3A%2F%2Fmailcow.github.io

spantaleev committed:
Fix a few more broken documentation links

spantaleev committed:
Fix broken documentation links in translation files

spantaleev commented:
I believe I've fixed all other broken documentation links, but please check if I've missed anything!

I've updated the translation files in [5c4440e](https://github.com/mailcow/mailcow-dockerized/pull/4458/commits/5c4440e9a4c0c25a20dd514090cecb4e5f889b16), but I'm not sure if I was supposed to update those manually or if they're pulled from some translation system, etc.

MAGICCC commented:
Looks fine thanks!
We do have a translation tool, but it will pull the latest code anyway
https://translate.mailcow.email/engage/mailcow-dockerized/
